tests/integration.rs

```diff
@@ -120,6 +120,25 @@ fn invert() -> Result<()> {
   assert_output_eq(&["resize:1:1", "all"], "1")
 }
 
+#[test]
+fn save() -> Result<()> {
```

Comment:
Maybe we should refuse to save text images. It's not a very useful format, so I think we should treat it as a print format only.